### ISSUE TITLE
[Feature Request] Add iconColor and rightIconColor as options to QSelect items

### ISSUE THREAD

**Issue by nahuel** [quasarframework/quasar#913]
You can set the color of a `QIcon` by doing `<q-icon name="warning" color="negative"/>` but the `icon` and `rightIcon` color of a `QSelect` option can't be changed in the same way. It will be nice to do:

```html
<q-select :options="[{label : 'somelabel', 
                      value : 'someval',
                      icon  : 'warning',
                      iconColor : 'negative'   /***/
                     }]"/>`
```

**Comment by rstoenescu** [comment]
Available in edge and future v0.14.4.
Thanks for this!

**Comment by rstoenescu** [comment]
Props will be "leftColor" and "rightColor".
